src/components/MyKiva/BadgesSection.vue

```diff
@@ -95,6 +92,27 @@ const ctaCaption = badge => {
 	}
 	return badge.hasStarted ? 'Continue' : 'Start this journey';
 };
+
+const badgeClicked = badge => {
+	$kvTrackEvent(
+		'portfolio',
+		'click',
+		'Badge journey map',
+		badge.challengeName,
+		badge.level
+	);
+	emit('badge-clicked', badge);
+};
+
+watch(route, () => {
+	if (route?.hash === '#my-impact-journeys' && Object.keys(route?.query).length !== 0) {
```

Comment:
Where do we currently open a link with journey on the query string?

Reply:
This is only case for my kiva page, coming from hero carousel.

Reply:
@dyersituations

Reply:
Sorry just got to the airport 😄. That makes sense!

Reply:
That also makes sense ✈️, safe travels!